### DISCUSSION TITLE
astradb error in langflow

### DISCUSSION THREAD

**Discussion by cnvrt** [langflow-ai/langflow#3890] [General]
everytime showing this error for every model

all api keys working

this error showing in Astra DB of Vector Store

![Langflow-Vector_Store-AstraDB-error](https://github.com/user-attachments/assets/66bcae6e-7348-405c-b4c0-f4f1fd032d33)

Error Building Component
Error building Component Astra DB: Error initializing AstraDBVectorStore: Error code: 400 - {'error': {'code': 'OperationNotSupported', 'message': 'The embeddings operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993.'}}

**Comment by carlosrcoelho** [comment]
@cnvrt 
In your Embedding component, you're using the gpt-35-turbo model, but it's necessary to use an Embedding model, such as:

- text-embedding-3-small

- text-embedding-3-large

- text-embedding-ada-002
